content/200-concepts/200-database-connectors/07-mongodb.mdx

```diff
@@ -188,3 +188,55 @@ To resolve this, we suggest you change your deployment to one with a replica set
 One simple way for this is to use [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) to launch a free instance that has replica set support out of the box.
 
 There's also an option to run the replica set locally with this guide: https://docs.mongodb.com/manual/tutorial/convert-standalone-to-replica-set
+
+## Type mapping between MongoDB and the Prisma schema
+
+> Alternatively, see the [Prisma schema reference](/reference/api-reference/prisma-schema-reference#model-field-scalar-types) for type mappings organized by Prisma type.
+
+### Mapping from Prisma to MongoDB types on migration
```

Comment:
```suggestion
### Mapping from Prisma to MongoDB types on Migrations
```

Reply:
Hm, this sounds odd to me. It's Prisma Migrate but we normally talk about 'migrations', lower case, elsewhere in docs